src/main/java/uk/gov/hmcts/reform/pip/account/management/database/UserRepository.java

```diff
@@ -17,4 +17,9 @@ List<PiUser> findExistingByProvenanceId(@Param("provUserId") String provenanceUs
                                             @Param("userProv") String userProvenance);
 
     Optional<PiUser> findByUserId(UUID userId);
+
+    @Query(value = "SELECT * FROM pi_user WHERE email=:email AND user_provenance=:userProv",
+        nativeQuery = true)
+    Optional<PiUser> findByEmailAndProvenanceId(@Param("email") String provenanceUserId,
```

Comment:
not really provenenaceUserId, its email, those are 2 very different values, careful not to cause confusion doing this. would name appropiately